orchestrator/db/sorting/sorting.py

```diff
@@ -16,9 +16,10 @@
 
 from more_itertools import partition
 from pydantic import BaseModel
+from sqlalchemy import Column
 from sqlalchemy.sql import expression
 
-from orchestrator.db.database import BaseModel as SqlBaseModel
+from orchestrator.api.error_handling import ProblemDetailException
 from orchestrator.db.database import SearchQuery
 from orchestrator.db.filters import CallableErrorHander
```

Comment:
I missed it the few first times reading this, but there's a typo 
```suggestion
from orchestrator.db.filters import CallableErrorHandler
```